server/service/src/stocktake/insert/generate.rs

```diff
@@ -121,8 +137,8 @@ fn generate_stocktake_lines(
                          barcode_id: _,
                          item_variant_id,
                          donor_link_id,
-                         vvm_status_id: _,
-                         campaign_id: _,
+                         vvm_status_id: _, // Todo?
+                         campaign_id: _,   // Todo?
```

Comment:
Should we be pulling campaigns into stocktake, I feel like we probably should?

Reply:
Yes, that's in 2.10 probably